molgenis-data-mapper/src/main/java/org/molgenis/data/mapper/service/impl/MappingServiceImpl.java

```diff
@@ -183,14 +185,22 @@ public String applyMappings(MappingTarget mappingTarget, String entityName, bool
 		}
 
 		Repository<Entity> targetRepo;
-		if (!dataService.hasRepository(entityName))
+		if (!dataService.hasRepository(entityName) && !dataService.hasRepository("base_"+entityName))
```

Comment:
no because an entity can be in another package as well, it does not have to be in base